packages/capabilityStatement/src/capabilityStatement.ts

```diff
@@ -47,4 +47,4 @@ export const handler = middy(lambdaHandler)
       }
     })
   )
-  .use(errorHandler({logger}))
+  .use(errorHandler({logger: console}))
```

Comment:
Why did we change this?

Reply:
The value `logger` has been assigned to the property `logger`
`.use(errorHandler({logger: logger}))`